payments-core/src/main/java/com/stripe/android/paymentsheet/model/SupportedSavedPaymentMethod.kt

```diff
@@ -0,0 +1,71 @@
+package com.stripe.android.paymentsheet.model
+
+import android.content.res.Resources
+import androidx.annotation.DrawableRes
+import com.stripe.android.R
+import com.stripe.android.model.CardBrand
+import com.stripe.android.model.PaymentMethod
+
+/**
+ * Enum defining all types of saved payment method supported on Payment Sheet.
+ *
+ * These are fetched from the
+ * [PaymentMethods API endpoint](https://stripe.com/docs/api/payment_methods/list) for returning
+ * customers.
+ */
+internal enum class SupportedSavedPaymentMethod(
```

Comment:
Does this need to be a dedicated `enum`? Can it be just a `Map` of supported types? that way we won't need to redefine `#fromCode`

Reply:
It doesn't, it can be a map.